react/src/components/ImageList.tsx

```diff
@@ -590,22 +528,13 @@ const ImageList: React.FC<{ style?: React.CSSProperties }> = ({ style }) => {
               }
             },
           })}
-          showSorterTooltip={false}
-        />
-        {/* <Flex
-          justify="end"
-          style={{
-            padding: token.paddingXXS,
+          order={queryParams.order}
+          onChangeOrder={(newOrder) => {
+            startPageChangeTransition(() => {
+              setQuery({ order: newOrder }, 'replaceIn');
+            });
           }}
-        >
-          <Button
-            type="text"
-            icon={<SettingOutlined />}
-            onClick={() => {
-              toggleColumnSettingModal();
-            }}
-          />
-        </Flex> */}
+        />
```

Comment:
Just asking for feedback, currently we reset and connect pagination when filtering changes, should this also be done for ordering?